backend/site-api/src/db/sites_metadata_repo.rs

```diff
@@ -137,7 +137,7 @@ impl SitesMetadataRepoTrait for SitesMetadataRepo {
         let (mut query, count) = append_comma(query, "disabled", dto.disabled, count);
 
         if count == 0 {
-            return Err(DbError::NoUpdate);
+            return Ok(());
```

Comment:
I changed it here to return OK instead of an error to avoid having too many complex conditions like dto.disabled.is_some() &&... in `update_site_metadata.rs`.

Reply:
Oh, sorry for overlooking that. I'll make sure to avoid that next time!

Reply:
No problem! I think your version is better, the API is not used that frequently so simpler/lower maintenance code is preferred.